README.md

```diff
@@ -0,0 +1,5 @@
+Temp readme
+
+tech stack: js,html, css
+
```

Comment:
```jsx

export const useCloseFromEsc = (setter) => {
    useEffect(() => {
        const closeFromEsc = (e) => e.key === "Escape" && setter(false);
        document.addEventListener("keydown", closeFromEsc);
    }, [setter]);
};

```


you need to remove the listener in the `clean up` function in `useEffect` to clear the listener on unmount



```jsx


export const useCloseFromEsc = (setter) => {
  useEffect(() => {
    const closeFromEsc = (e) => e.key === "Escape" && setter(false);
    document.addEventListener("keydown", closeFromEsc);
    return () => {
      document.removeEventListener("keydown", closeFromEsc);  // this is the clean up
    };
  }, [setter]);
};

```